app/models/post.rb

```diff
@@ -5,6 +5,10 @@ class Post < ApplicationRecord
 
   after_create :update_user_posts_counter
 
+  validates :title, presence: true, length: { in: 3..250 }
+  validates :comments_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }, allow_nil: true
+  validates :likes_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }, allow_nil: true
```

Comment:
[OPTIONAL]
- [ ] I would suggest not accepting a `nil` value to your variables as they cause all sorts of bugs in your program, Kindly you may consider removing that. Rather you may consider initializing these variables (like, comment_counter) to zero

Reply:
Thank you for the suggestions. I have implemented them. 👍🏾